tenets/codelingo/code-review-comments/named-result-params/main.go

```diff
@@ -0,0 +1,44 @@
+package somepackage
+
+// Thing is an example
+type Thing struct {
+	Name   string
+	Number int
+}
+
+// Example is an example
+type Example struct {
+	Count    int
+	TheThing *Thing
+}
+
+// GetThing returns the Example's Thing
+func (e *Example) GetThing() (thing *Thing) {
```

Comment:
```suggestion
func (e *Example) GetThing() (thing *Thing) { // ISSUE
```

Makes it easier to read.